swagger-api/swagger-ui
# fix: highlight code scroll console error (#7497)

### Description

* fix for "Unable to preventDefault inside passive event listener due to target being treated as passive" 
  * commonly advised solution => setting CSS property `touch-action: none` did not fix the issue
  * as a workaround for the error manual event listener for scroll has been added to componentDidMount to be able to set option {passive:false} 




### Motivation and Context
Fixes #7497 








### How Has This Been Tested?



* go to `swagger-ui` and run `npm run dev` wait a while and open `localhost:3200`
  *   expand `PUT /pets` endpoint
  *   click inspect on the example body
    * in dev tools - elements search for `code` element within div with class "highlight-code"
      * edit `code` element as HTML and duplicate it => as a result the example body will be longer and scrollable
      * now that the example body is scrollable open `console in devtools` and scroll example body all the way till the whole page starts scrolling => there should be no errors in the console
      
    



### Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/47686623/134187601-8639a928-b006-43e7-ae7d-d42a65d09757.png)



## Checklist



### My PR contains... 


### My changes...

### Documentation

### Automated tests
